### PR TITLE
enabled owners-label/verify-owners plugin for etcd-io

### PR DESCRIPTION
**Pull request by liangyuanpeng**
part of https://github.com/etcd-io/etcd/issues/17970
/assign @jmhbnz

### CODE REVIEW

File: config/prow/plugins.yaml
Comment (liangyuanpeng) [inline config/prow/plugins.yaml:1176]: This is we really haved so i put they here and `owners-label` is a new plugin for auto add label. Maybe we also want to add the plugin for `lgtm` and `approve`

```yaml
    - welcome
    - assign
    - trigger
    - wip
```

**Review by jmhbnz — APPROVED** [review]
Thanks for picking this up @liangyuanpeng 
/lgtm

**Comment by jmhbnz** [comment]
Based on the failure it looks like we would also need `verify-owners`?

> {"component":"checkconfig","file":"sigs.k8s.io/prow/cmd/checkconfig/main.go:91","func":"main.reportWarning","level":"warning","msg":"the following orgs or repos enable at least one plugin that uses OWNERS files (approve, blunderbuss, owners-label) but do not enable the verify-owners plugin to ensure validity of OWNERS files: [org: etcd-io]","severity":"warning","time":"2024-05-09T08:07:54Z"}

Ref: https://prow.k8s.io/plugins

> The verify-owners plugin validates OWNERS and OWNERS_ALIASES files (by default) and ensures that they always contain collaborators of the org, if they are modified in a PR.

**Comment by liangyuanpeng** [comment]
@jmhbnz  is a approver of etcd,so lgtm is engough (I think?).  

 looking for approval for this file, @dims ,Thanks :)

**Pull request merged**
